### PR TITLE
React Data Insertion

### PR DESCRIPTION
**Pull request by davidislam**
Screen Shot 2020-06-12 at 7 46 32 PM

#8 The purpose of this ticket was to create a form in the FE using `react-bootstrap` and connect it to the backend server so that a new entry can be made in the db whenever the form is submitted.

In the future I want to add user feedback such as warning messages telling the user input (like textarea) is invalid.

### CODE REVIEW

File: Frontend/src/serverCalls.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    .post(url, entry)
+    .then(res => {
+      console.log(res.data)
+    })
+    .catch(res => {
+      console.log(res.data)
+    })
+}
+
+
```
Comment (danielanatolie) [inline Frontend/src/serverCalls.js:14]: Please have 1 space before every function not 2.

File: Frontend/src/App.js [hunk 2]
```diff
@@ -1,26 +1,95 @@
...
+      expiry_length: '',
+      text_message: ''
+    });
+  }
+
+  render() {
+    const { url, user_id, expiry_length, text_message } = this.state;
+    return (
+      <div className="App">
+
```
Comment (danielanatolie) [inline Frontend/src/App.js:44]: There shouldn't be a space between <div>s

Comment (davidislam) [reply Frontend/src/App.js]: f0283cb

File: Frontend/src/serverCalls.js (hunk 1 above)
Comment (davidislam) [reply Frontend/src/serverCalls.js]: f0283cb

**Pull request merged**